troubleshooting/debugging.md

```diff
@@ -235,10 +235,10 @@ There are some caveats when using Valgrind. Its a debugging tool, and so it sign
 
 ##### Option 1: Build from prod release (Easier but less robust)
 
-Use the Dockerfile below to create a new container image that will invoke your Fluent Bit version using Valgrind. Replace the "builder" image with whatever AWS for Fluent Bit release you are using, in this case, we are testing 2.21.0. The Dockerfile here will copy the original binary into a new image with Valgrind. Valgrind is a little bit like a mini-VM that will run the Fluent Bit binary and inspect the code at runtime. When you terminate the container, Valgrind will output diagnostic information on shutdown, with summary of memory leaks it detected. This output will allow us to determine which part of the code caused the leak. Valgrind can also be used to find the source of segmentation faults
+Use the Dockerfile below to create a new container image that will invoke your Fluent Bit version using Valgrind. Replace the "builder" image with whatever AWS for Fluent Bit release you are using, in this case, we are testing 2.28.0. The Dockerfile here will copy the original binary into a new image with Valgrind. Valgrind is a little bit like a mini-VM that will run the Fluent Bit binary and inspect the code at runtime. When you terminate the container, Valgrind will output diagnostic information on shutdown, with summary of memory leaks it detected. This output will allow us to determine which part of the code caused the leak. Valgrind can also be used to find the source of segmentation faults
 
 ```
-FROM public.ecr.aws/aws-observability/aws-for-fluent-bit:2.21.0 as builder
+FROM public.ecr.aws/aws-observability/aws-for-fluent-bit:2.28.0 as builder
```

Comment:
I see... I don't think this is a good long term solution since we will need to keep updating it. I'd prefer to put `latest` or `stable` in the example.

Reply:
Ok, I'll update it.

Reply:
I went with `latest` since that's the one I suspect people will most likely be debugging.